packages/dear-imgui/examples/node-example/src/index.tsx

```
@@ -1,19 +1,20 @@
+import { resolve } from "path";
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)